Anlab.Mvc/Views/Shared/_OrderDetails.cshtml

```diff
@@ -111,7 +120,17 @@
                     <i class="analysisTooltip fa fa-info-circle" onshow="renderTitle('@temp.Notes')" title="@temp.Notes" aria-hidden="true" data-toggle="tooltip" data-html="true"></i>
                 }
             </td>
-            <td>@test.Id</td>
+            <td>
+                @if(Model.HideLabDetails)
+                {
+                    <a href="/analysis/@temp.Category/@temp.Sop" target="_blank">@temp.Sop</a>

```

Comment:
@laholstege Use this instead:
`<a asp-controller="Analysis" asp-action="Details" asp-route-id="@temp.Sop" target="_blank">@temp.Sop</a>`